### PR TITLE
WD-7205 - enrich /real-time page with images

### PR DESCRIPTION
**Pull request by lizzochek**
## Done

- Added images in https://ubuntu-com-13359.demos.haus/real-time

## QA

- [figma design](https://www.figma.com/file/Cr1S07Ivk632jniBAG89S8/23.10-U.com---Real-time-Ubuntu?type=design&node-id=0-1&mode=design&t=QcxshpHOoI7PE0oV-0)
- [demo link](https://ubuntu-com-13359.demos.haus/)
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
-  Check that images and text are correct

## Issue / Card
[WD-7205](https://warthogs.atlassian.net/browse/WD-7205)
Fixes #

## Help

[WD-7205]: https://warthogs.atlassian.net/browse/WD-7205?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by lyubomir-popov** [comment]
hi,

- the images are low resolution - please use the high definition flag (@petesfrench recently used it) when requesting images from the cdn. the original 3x images are here in case you need them https://drive.google.com/drive/folders/1RJ1yVauVlbI5wzY1mgoSCkUkQ4dy0rS0?usp=drive_link
- please add the 0.03% dark opacity rectangle backgrounds on a parentr div of the illustrations as in the design.
- please hide them on mobile (small breakpoint only)

**Comment by lyubomir-popov** [comment]
hi, one final thing - min-height doesn't work well. Can we please have a css grid mini component here? we can then upstream it.

image

**Comment by juanruitina** [comment]
Images are decorative, so please use an empty alt tag. Other than that, LGTM.

File: templates/real-time/index.html [hunk 1]
```diff
@@ -59,7 +59,89 @@ <h2>Why use Real-time Ubuntu?</h2>
...
+    <div class="col-12 col-medium-3 p-section--shallow">
+      <h2>Real-time Ubuntu use cases</h2>
+    </div>
+  </div>
+  <div class="row u-hide--medium u-equal-height">
+    <div class="col-3 col-medium-3">
+      <h3 class="p-heading--5">Industrial settings</h3>
+      <p class="p-cases">PLCs in assembly lines must deliver and process data in real-time to ensure system
+        integrity and continuous
+        production.</p>
```
Comment (akbarkz) [inline templates/real-time/index.html:73]: please combine this with the previous line as it fits there

File: templates/real-time/index.html [hunk 2]
```diff
@@ -59,7 +59,89 @@ <h2>Why use Real-time Ubuntu?</h2>
...
+        loading="lazy",
+        ) | safe
+        }}
+      </div>
+    </div>
+  </div>
+  <div class="row u-hide--medium">
+    <p>Not sure real-time is for you? <br /><a href="/kernel/real-time/contact-us">Get in touch</a> to talk over
+      your
+      use case with our sales team</p>
```
Comment (akbarkz) [inline templates/real-time/index.html:142]: and here, please combine it with the prev line

**Comment by akbarkz** [comment]
@lizzochek the demo link is not opening, can you please re-generate it?

**Review by akbarkz — APPROVED** [review]

**Pull request merged**